private/buf/bufconvert/bufconvert.go

```diff
@@ -30,10 +30,16 @@ import (
 const (
 	// MessageEncodingBin is the binary image encoding.
 	MessageEncodingBin MessageEncoding = iota + 1
+	// MessageEncodingText is the text image encoding.
+	MessageEncodingText
 	// MessageEncodingJSON is the JSON image encoding.
 	MessageEncodingJSON
 	// formatBin is the binary format.
 	formatBin = "bin"
+	// formatBinProto is also used for binary proto format.
+	formatBinProto = "binproto"
+	// formatTextProto is the proto text format.
+	formatTextProto = "textproto"
```

Comment:
`formatText = "text"`

Reply:
I do not feel comfortable saying all .bin or .text files are proto encoded. Not do I want people to have to specify #format= when they are using more explicit and common files extensions.